spec/Bex/Behat/Context/TestRunnerContextSpec.php

```diff
@@ -49,6 +49,16 @@ function it_creates_temporary_working_directory(Filesystem $filesystem)
         $this->createWorkingDirectory();
     }
 
+    function it_removes_temporary_working_directory(Filesystem $filesystem)
+    {
+        $tempDir = sys_get_temp_dir() .'/behat-test-runner';
```

Comment:
It's surprising that this spec passes as the temporary working directory is created by
`tempnam(sys_get_temp_dir(), 'behat-test-runner') === "/tmp/behat-test-runner<random-characters>" !== "/tmp/behat-test-runner"`.

Reply:
not surprising because `$filesystem->remove(Argument::containingString($tempDir))->shouldBeCalled();` - see `Argument::containingString` - you wrote it! :D

Reply:
It's too early morning to me. Sorry.